forge-gui/res/cardsfolder/upcoming/relic_of_legends.txt

```diff
@@ -0,0 +1,8 @@
+Name:Relic of Legends
+ManaCost:3
+Types:Artifact
+A:AB$ Mana | Cost$ T | Produced$ Any | SpellDescription$ Add one mana of any color.
+A:AB$ Mana | Cost$ tapXType<1/Creature.Legendary> | Produced$ Any | SpellDescription$ Tap an untapped legendary creature you control: Add one mana of any color.
```

Comment:
Does this cost look OK in the Card Detail pane?

Reply:
good catch, I always forget to remove the cost from the description